gui/widget.cpp

```diff
@@ -451,23 +426,6 @@ void PicButtonWidget::setGfx(const Graphics::Surface *gfx, int statenum) {
 	_gfx[statenum].copyFrom(*gfx);
 }
 
-void PicButtonWidget::setAGfx(const Graphics::TransparentSurface *gfx, int statenum, ThemeEngine::AutoScaleMode mode) {
```

Comment:
Why is this removed? It is not used in the current code, but I added transparent PNG usage to the themes, and it could be used for the buttons now.

Reply:
I removed that code because:
* It was never used since its introduction 4 years ago
* It was not updated to work with the clipping rect (meaning it is broken)

The same reasoning applies to your other comments.

Reply:
Valid point. Then it is fine to merge, and once I work on real adding of the PNG, I can re-add those.

Still, in my opinion, these two commits need to be separated, as this has nothing to do with the unification.